bootstrap/sql/oracle/v001__create_table.sql

```diff
@@ -0,0 +1,96 @@
+  "compatibility"   VARCHAR2(255)          NOT NULL,
+  "validationLevel" VARCHAR2(255)          NOT NULL, -- added in 0.3.1, table should be altered to add this column from earlier versions.
+  "description"     VARCHAR2(4000),
+  "evolve"          NUMBER(1)              NOT NULL,
+  "timestamp"       NUMBER(19,0)           NOT NULL,
+  CONSTRAINT schema_metadata_info_pk PRIMARY KEY ("name"),
+  CONSTRAINT schema_metadata_info_uk UNIQUE ("id")
+);
+
+CREATE TABLE "schema_version_info" (
+  "id"               NUMBER(19,0)          NOT NULL,
+  "description"      VARCHAR2(4000),
+  "schemaText"       CLOB                  NOT NULL,
+  "fingerprint"      VARCHAR2(4000)        NOT NULL,
+  "version"          NUMBER(10,0)          NOT NULL,
+  "schemaMetadataId" NUMBER(19,0)          NOT NULL,
+  "timestamp"        NUMBER(19,0)          NOT NULL,
+  "state"            NUMBER(3,0)           NOT NULL,
+  "name"             VARCHAR2(255)         NOT NULL,
+  CONSTRAINT schema_vinfo_uk_id UNIQUE ("id"),
+  CONSTRAINT schema_vinfo_uk_mid_v UNIQUE ("schemaMetadataId", "version"),
+  CONSTRAINT schema_vinfo_pk PRIMARY KEY ("name", "version"),
+  CONSTRAINT schema_vinfo_fk_smi_id FOREIGN KEY ("schemaMetadataId") REFERENCES "schema_metadata_info" ("id") ON DELETE CASCADE,
+  CONSTRAINT schema_vinfo_fk_smi_name FOREIGN KEY ("name") REFERENCES "schema_metadata_info" ("name") ON DELETE CASCADE
+);
+
+CREATE TABLE "schema_field_info" (
+  "id"               NUMBER(19,0)          NOT NULL,
+  "schemaInstanceId" NUMBER(19,0)          NOT NULL,
+  "timestamp"        NUMBER(19,0)          NOT NULL,
+  "name"             VARCHAR2(255)         NOT NULL,
+  "fieldNamespace"   VARCHAR2(255),
+  "type"             VARCHAR2(255)         NOT NULL,
+  CONSTRAINT schema_field_info_pk PRIMARY KEY ("id"),
+  CONSTRAINT schema_field_info_fk_svid FOREIGN KEY ("schemaInstanceId") REFERENCES "schema_version_info" ("id") ON DELETE CASCADE
+);
+
+CREATE TABLE "schema_serdes_info" (
+  "id"                    NUMBER(19,0)          NOT NULL,
+  "description"           VARCHAR2(4000),
+  "name"                  VARCHAR2(4000)        NOT NULL,
+  "fileId"                VARCHAR2(4000)        NOT NULL,
+  "serializerClassName"   VARCHAR2(4000)        NOT NULL,
+  "deserializerClassName" VARCHAR2(4000)        NOT NULL,
+  "timestamp"             NUMBER(19,0)          NOT NULL,
+  CONSTRAINT schema_serdes_info_pk PRIMARY KEY ("id")
+);
+
+CREATE TABLE "schema_serdes_mapping" (
+  "schemaMetadataId" NUMBER(19,0) NOT NULL,
+  "serDesId"         NUMBER(19,0) NOT NULL,
+  CONSTRAINT schema_serdes_mapping_pk PRIMARY KEY ("schemaMetadataId", "serDesId")
+);
+
+CREATE TABLE "schema_version_state" (
+  "id"              NUMBER(19,0)          NOT NULL,
+  "schemaVersionId" NUMBER(19,0)          NOT NULL,
+  "stateId"         NUMBER(3,0)           NOT NULL,
+  "sequence"        NUMBER(10,0)          NOT NULL,
+  "timestamp"       NUMBER(19,0)          NOT NULL,
+  "details"         CLOB,
```

Comment:
This may need to be declared as BLOB instead of CLOB?

Reply:
Earlier the details column was mapped as String in Java, so I kept it as CLOB, will change this to BLOB.